cmd/influxd/inspect/verify_wal/verify_wal.go

```diff
@@ -0,0 +1,129 @@
+package verify_wal
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"text/tabwriter"
+	"time"
+
+	"github.com/influxdata/influxdb/v2/internal/fs"
+	"github.com/influxdata/influxdb/v2/tsdb/engine/tsm1"
+	"github.com/spf13/cobra"
+)
+
+type args struct {
+	dir string
+}
+
+func NewVerifyWALCommand() *cobra.Command {
+	var arguments args
+	cmd := &cobra.Command{
+		Use:   `verify-wal`,
+		Short: "Check for WAL corruption",
+		Long: `
+This command will analyze the WAL (Write-Ahead Log) in a storage directory to 
+check if there are any corrupt files. If any corrupt files are found, the names
+of said corrupt files will be reported. The tool will also count the total number
+of entries in the scanned WAL files, in case this is of interest.
+For each file, the following is output:
+	* The file name;
+	* "clean" (if the file is clean) OR 
+      The first position of any corruption that is found
+In the summary section, the following is printed:
+	* The number of WAL files scanned;
+	* The number of WAL entries scanned;
+	* A list of files found to be corrupt`,
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return arguments.Run(cmd)
+		},
+	}
+
+	dir, err := fs.InfluxDir()
+	if err != nil {
+		panic(err)
+	}
+	dir = filepath.Join(dir, "engine/wal")
+	cmd.Flags().StringVarP(&arguments.dir, "data-dir", "", dir, fmt.Sprintf("use provided data directory (defaults to %s).", dir))
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&arguments.dir, "wal-dir", "", dir, "use provided WAL directory.")
```
Since:
* It looks like `dir` will end up pointing to `~/.influxdbv2/engine/wal` by default, and the `data-dir` typically means `engine/data`
* Cobra will auto-append the default